Jenkinsfile

```diff
@@ -14,16 +14,22 @@ def getMergedPrNo() {
     return mergedPrNo ? "pr$mergedPrNo" : ''
 }
 
+def getRepoURL() {
+  return sh(returnStdout: true, script: "git config --get remote.origin.url").trim()
+}
+ 
+def getCommitSha() {
+  return sh(returnStdout: true, script: "git rev-parse HEAD").trim()
+}
+
```

Comment:
> Looks good to me. One thing I missed in the previous PR, which Mark pointed out in mine, is the service name in the jenkins-aws.yaml file. As we're using namespaces, there's no need to prefix the name in that file, so I've removed the "aws-" prefix from there.
> This can be done as part of the deployment issue though, as that involves a change to that file anyway. I'm happy for this to be merged.
> 
> UPDATE - Missed that the Jenkinsfile doesn't actually run due to duplicated method names.

Makes sense just to remove the prefix now so I've done that.  Also fixed the duplicate methods.